script/application.go

```diff
@@ -0,0 +1,198 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v60/github"
+)
+
+type Project struct {
+	Name         string `json:"name"`
+	Description  string `json:"description"`
+	Contributors int    `json:"contributors"`
+	HomeUrl      string `json:"home_url"`
+	RepoUrl      string `json:"repo_url,omitempty"`
+	LicenseType  string `json:"license_type,omitempty"`
+	LicenseUrl   string `json:"license_url,omitempty"`
+	IsEvent      bool   `json:"is_event"`
+	IsTeam       bool   `json:"is_team"`
+}
+
+type Applicant struct {
+	Name  string `json:"name"`
+	Email string `json:"email"`
+	Role  string `json:"role"`
+	Id    int64  `json:"id"`
+}
+
+type Application struct {
+	validator Validator         `json:"-"`
+	sections  map[string]string `json:"-"`
+	Problems  []error           `json:"-"`
+
+	Account     string    `json:"account"`
+	Project     Project   `json:"project"`
+	Applicant   Applicant `json:"applicant"`
+	CanContact  bool      `json:"can_contact"`
+	ApproverId  int       `json:"approver_id,omitempty"`
+	IssueNumber int       `json:"issue_number"`
+	CreatedAt   time.Time `json:"created_at"`
+}
+
+func (a *Application) Parse(issue github.Issue) {
```

Comment:
Is there a specific reason you chose to pass the issue in by value? If not, I suggest passing it as a pointer. It's a little more efficient as you aren't making a copy of the data (although this likely has negligible impact), but also the code ends up a little cleaner as you won't need to dereference it in the caller, i.e.:

this:
```r.application.Parse(*r.gitHub.Issue)```

becomes:
```r.application.Parse(r.gitHub.Issue)```